backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb

```diff
@@ -1,6 +1,6 @@
 module CeqrData
-  class ScaBluebook < Base  
-    self.schema = "sca_bluebook"
+  class CeqrSchoolBuildings < Base  
+    self.schema = "ceqr_school_buildings"
 
 		# all high schools (HS, ISHS) within project borough
 		def high_schools_in_boro(borocode)
```

Comment:
Do we need to also update the interfaces for High Schools and Intermediate Schools?

I like the name of this function `high schools_in_boro...`, and think that all three functions in this file should follow a similar naming (`primary_schools_in_subdistricts` and `intermediate_schools_in_subdisricts`).

Reply:
Good point